### PR TITLE
fix(repo-page): add value prop to select

### PR DESCRIPTION
**Pull request by trybick**
#### Summary

- Select component was not showing the value of the currently selected filter, this fixes that.
- Updates a meta tag which shows when posting URL in slack

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in semantic format `TYPE(scope): description`

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-    <div id="root"></div>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="/favicon.ico" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Explore GitHub Repositories and Commits without pagination." />
```
Comment (trybick) [inline public/index.html:9]: Everything here is formatting changes except line 9 "description"

File: src/components/repo-list/RepoList.js [hunk 2]
```diff
@@ -57,7 +57,12 @@ export const RepoList = ({ repos, user }) => {
 
   const SelectFilter = () => (
     <Flex justifyContent="flex-end" mb="12px">
-      <Select fontSize="15px" onChange={onFilterChange} w="160px">
+      <Select
+        fontSize="15px"
+        onChange={onFilterChange}
+        value={repoFilter}
```
Comment (trybick) [inline src/components/repo-list/RepoList.js]: adds value prop

**Review by davidholyko — APPROVED** [review]
LGTM

File: src/components/repo-list/RepoList.js (hunk 2 above)
Comment (davidholyko) [reply src/components/repo-list/RepoList.js]: small note, I dont think this Select is actually a filter in terms of what it does. maybe add a TODO we can rename later.

Our use of the word Filter for repo list is kind of just a sort

**Review by davidholyko — APPROVED** [review]

**Pull request merged**